applets/wncklet/window-list.c

```diff
@@ -25,6 +25,11 @@
 #include <libwnck/libwnck.h>
 #endif /* HAVE_X11 */
 
+#ifndef HAVE_X11
+#include <gdk/gdkwayland.h>
+#define GDK_IS_X11_DISPLAY(object)        !(G_TYPE_CHECK_INSTANCE_TYPE ((object), GDK_TYPE_WAYLAND_DISPLAY))
+#endif
+
```

Comment:
And again. I'm gonna stop complaining about this in the rest of the diff, it's the same everywhere.